database/repo.go

```diff
@@ -61,6 +65,7 @@ type Repo struct {
 	AllowTag     sql.NullBool   `sql:"allow_tag"`
 	AllowComment sql.NullBool   `sql:"allow_comment"`
 	PipelineType sql.NullString `sql:"pipeline_type"`
+	PreviousName sql.NullString `sql:"name_history"`
```

Comment:
Should this be `previous_name`?

```suggestion
	PreviousName sql.NullString `sql:"previous_name"`
```